src/Network/Network/help/New-AzVpnSiteLinkConnection.md

```diff
@@ -0,0 +1,235 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# New-AzVpnSiteLinkConnection
+
+## SYNOPSIS
+Creates an Azure VpnSiteLink object.
```

Comment:
Since it's a doc for VpnSiteLinkConnection, it should be "Azure VpnSiteLinkConnection object"

Reply:
done